src/components/pages/CaseOverview/CaseDetails.js

```diff
@@ -34,7 +41,7 @@ const CaseDetails = props => {
           >
             <span>Ok</span>
           </AntDButton>
-          {localStorage.role === 'admin' || '' ? (
+          {localStorage.role_name === 'admin' || '' ? (
```

Comment:
Another good catch here, ensuring the proper variable is being checked to ensure only Admins have edit permissions. Not a merge-blocker considering the scope of this PR, but we may want to revisit this usage wholistically to make it more secure. If a user who isn't an admin was savvy enough to check their localStorage they could easily give themselves edit permissions, so we should use the role IDs in our database to check for Admin permissions as opposed to a plaintext name as a first step toward being more secure.